packages/design-system/src/components/icon-button.tsx

```diff
@@ -23,7 +23,7 @@ export const IconButton = styled("button", {
   "&::before, &::after": {
     boxSizing: "border-box",
   },
-  "&:hover, &:active": {
+  "&:focus, &:hover, &:active": {
```

Comment:
I'd rather duplicate the rule in the next selector. Now it is much harder to understand what are the resulting styles going to be.